tardis/adapters/sites/moab.py

```diff
@@ -121,11 +122,7 @@ async def deploy_resource(
         self, resource_attributes: AttributeDict
     ) -> AttributeDict:
         request_command = (
-            f"msub -j oe -m p -l "
-            f"walltime={self.machine_type_configuration.Walltime},"
-            f"mem={self.machine_meta_data.Memory}gb,"
-            f"nodes={self.machine_type_configuration.NodeType} "
-            f"{self._startup_command}"
+            f"msub -j oe -m p {self.msub_cmdline_options()} {self._startup_command}"
```

Comment:
Actually I meant to add all options to the `AttributeDict` not only `-l`. Sorry, for not being precise enough.

Reply:
Sorry, this was my fault. I just forgot about the other options.